product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/performing_a_graphical_installation_on_windows.mdx

```diff
@@ -85,26 +88,27 @@ After selecting the components you wish to install, click `Next` to open the `Ad
 By default, the EDB Postgres Advanced Server `data` files are saved to:
 
 ```text
-C:\Program Files\edb\as15\data
+C:\Program Files\edb<*xx*>\data
 ```
+Where <*xx*> represents the EDB Postgres Advanced Server version number.
 
 The default location of the EDB Postgres Advanced Server `Write-Ahead Log (WAL) Directory` is:
 
 ```text
-C:\Program Files\edb\as15\data\pg_wal
+C:\Program Files\edb\as<*xx*>\data\pg_wal
 ```
 
 EDB Postgres Advanced Server uses write-ahead logs to promote transaction safety and speed transaction processing; when you make a change to a table, the change is stored in shared memory and a record of the change is written to the write-ahead log. When you perform a `COMMIT`, EDB Postgres Advanced Server writes contents of the write-ahead log to disk.
 
-Accept the default file locations, or use the `File Browser` icon to select an alternate location; click `Next` to continue to the `EDB Postgres Advanced Server Dialect` window.
+Accept the default file locations, or use the `File Browser` icon to select an alternate location; click `Next` to continue to the `EPAS Dialect` window.
 
 ![The EDB Postgres Advanced Server Dialect window](../../../images/advanced_server_dialect.png)
 
 <div style="text-align: center"> Fig. 6: The EDB Postgres Advanced Server Dialect window </div>
 
-Use the drop-down listbox on the `EDB Postgres Advanced Server Dialect` window to choose a server dialect. The server dialect specifies the compatibility features supported by EDB Postgres Advanced Server.
+Use the drop-down listbox on the `EDB Postgres Advanced Server Dialect` window to choose a server dialect. The server dialect specifies the compatibility features supported by EPAS.
```

Comment:
This PR has some EDB Postgres Advanced Server > EPAS changes also.

Reply:
Gotcha, thanks!